src/audio/kpb.c

```diff
@@ -543,8 +652,6 @@ struct comp_driver comp_kpb = {
 	},
 };
 
-void sys_comp_kpb_init(void);
```

Comment:
Why this was moved to the beginning? In another PR we have discussed placement of declarations init component functions to be just before the definition, because they are here only to meet the compiler expectations.